os/android/utils_android.h

```diff
@@ -34,6 +34,22 @@
 
 #include "hwcbuffer.h"
 
+#define HWC_UNUSED(x) ((void)&(x))
+#define ALWAYS_INLINE __attribute__((always_inline))
+#define NEVER_INLINE __attribute__((noinline))
+
+/**
+ */
+class NonCopyable {
+ protected:
```

Comment:
Instead of having this separate, we can  just mark the appropriate class functions directly to be the same:
private:
+  NonCopyable(NonCopyable const &) = delete;
+  void operator=(NonCopyable const &) = delete;